core/iwasm/interpreter/wasm_runtime.h

```diff
@@ -444,6 +446,14 @@ wasm_set_exception_with_id(WASMModuleInstance *module_inst, uint32 id);
 const char *
 wasm_get_exception(WASMModuleInstance *module);
 
+/**
+ * @brief Copy exception in buffer passed as parameter. Thread-safe version of
```

Comment:
Had better mention that the size of exception_buf must be no smaller than EXCEPTION_BUF_LEN.